charts/node/templates/statefulset.yaml

```diff
@@ -604,6 +607,10 @@ spec:
                 {{- end }}
                 {{- if and .Values.node.isParachain (not .Values.node.externalRelayChain.enabled) }}
                 -- \
+                {{- if  .Values.node.lightClientMode.enabled }}
+                --chain={{ if .Values.node.lightClientMode.relayChainCustomChainspecUrl }}{{ .Values.node.lightClientMode.relayChainCustomChainspecPath }}{{ else }}{{.Values.node.lightClientMode.relayChain}}{{ end }}
```

Comment:
we already defined `--chain=` flag twice (before and after `--`), defining it third time may introduce some confusions. 

I believe you can use just this:
```
{{ if .Values.node.lightClientMode.relayChainCustomChainspecUrl }}
--chain={{ .Values.node.lightClientMode.relayChainCustomChainspecPath }}
{{ end }}
```

If chain is not defined the substrate will take value from parachain chainspec [relay_chain](  https://github.com/paritytech/chainspecs/blob/main/rococo/parachain/rockmine2/chainspec-nonraw.json#L18 ) key

Reply:
We should retain the additional `--chain` for now since we're not sure if it has the same behavior of a regular collator mode since it uses light client.